test/e2e/data/cni/kindnet/kindnet.yaml

```diff
@@ -78,7 +78,7 @@ spec:
                 fieldRef:
                   fieldPath: status.podIP
             - name: POD_SUBNET
-              value: "192.168.0.0/16"
+              value: "fd00:100:96::/48"
```

Comment:
I think we should use variable substitution here, so kindnet manifest will adapt to what is being used for the cluster templates.
However I need to check if the machinery for CNI plugins support this and eventually make this possible (it is following a different code path due to some problems found in Azure due to a super long CNI yaml)

Reply:
resolving because I have summarised consideration about ipv6 E2E testing down in a comment